cpu/stm32/Makefile.dep

```diff
@@ -8,4 +8,11 @@ ifneq (,$(filter periph_usbdev,$(FEATURES_USED)))
   USEMODULE += xtimer
 endif
 
+# default to the Mask-ROM bootloader if no other bootloader is used
```

Comment:
I would completely remove the whole block and add
```
FEATURES_REQUIRED += bootloader_stm32
USEMODULE += usb_board_reset
```

in the test application. There's no need to perform this kind of dependency resolution at CPU level.